process/errors.go

```diff
@@ -576,8 +576,8 @@ var ErrHeaderIsBlackListed = errors.New("header is black listed")
 // ErrNilEconomicsData signals that nil economics data has been provided
 var ErrNilEconomicsData = errors.New("nil economics data")
 
-// ErrNilAlteredAccountsProviders signals that a nil altered accounts provider has been given
-var ErrNilAlteredAccountsProvider = errors.New("nil altered accounts provider")
+// ErrNilOutportDataProvider signals that a nil outport provided has been provided
```

Comment:
`// ErrNilOutportDataProvider signals that a nil outport data provider has been given`

Reply:
done